src/i18n-module.php

```diff
@@ -223,9 +223,9 @@ private function promo_message() {
 		$message = false;
 
 		if ( $this->translation_exists && $this->translation_loaded && $this->percent_translated < 90 ) {
-			$message = __( 'As you can see, there is a translation of this plugin in %1$s. This translation is currently %3$d%% complete. We need your help to make it complete and to fix any errors. Please register at %4$s to help complete the translation to %1$s!', $this->textdomain );
+			$message = __( 'As you can see, there is a translation of this plugin in %1$s. This translation is currently %3$d%%% complete. We need your help to make it complete and to fix any errors. Please register at %4$s to help complete the translation to %1$s!', $this->textdomain );
```

Comment:
It seems that `%3$d%%` splits up into `%3$d` and `%%`, which is already an escaped `%`.